src/index.html

```diff
@@ -1,13 +1,274 @@
+    <header class="header">
+      <div class="mobile-bg"></div>
+      <div class="header__top">
+        <a href="tel:+1 234 555-55-55" class="header__phone">+1 234 555-55-55</a>
+        <a href="tel:+1 234 555-55-55" class="header__logo--tel">
+          <img
+            src="images/tel.svg"
+            alt="phone"
+            class="header__logo--tel"
+          >
+        </a>
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="header__logo"
+        >
+        <a href="tel:+1 234 555-55-55"          class="header__call-to-order">call to order</a>
+        <button class="hamburger hamburger--collapse header__burger" type="button">
+          <span class="hamburger-box">
+            <span class="hamburger-inner"></span>
+          </span>
+        </button>
+        <!-- <img
+          src="images/burger.svg"
+          alt="burger"
+          class="header__burger"
+        > -->
+      </div>
+      <h1 class="header__title">Miami<br>Condo Kings</h1>
+      <nav class="header__nav nav">
+        <ul class="nav__container" id="nav__container">
+          <li class="nav__item">
+            <a href="#" class="nav__item--link active">Home</a>
+          </li>
+          <li class="nav__item">
+            <a href="#about_us" class="nav__item--link">About Us</a>
+          </li>
+          <li class="nav__item">
+            <a href="#gallery" class="nav__item--link">Gallery</a>
+          </li>
+          <li class="nav__item">
+            <a href="#consultation" class="nav__item--link">Consultation</a>
+          </li>
+          <li class="nav__item">
+            <a href="#testimonials" class="nav__item--link">Testimonials</a>
+          </li>
+          <li class="nav__item">
+            <a href="#contacts" class="nav__item--link ">Contacts</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <section class="main__section info">
+        <h2 class="info__title title--big" id="about_us">Miami best<br>Real estate <a href="https://www.google.com/maps/place/Miami,+FL,+USA/@25.7823404,-80.3695441,11z/data=!3m1!4b1!4m5!3m4!1s0x88d9b0a20ec8c111:0xff96f271ddad4f65!8m2!3d25.7616798!4d-80.1917902" class="link">here</a></h2>
+        <article class="info__article article">
+          <h4 class="article__title title--small">Over 50 years experience in love.</h4>
+          <p class="article__text">Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.</p>
+          <p class="article__text">The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists. Working directly with developers - so have more useful information about the profitable deals.</p>
```

Comment:
```suggestion
          <p class="article__text">
            The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists. Working directly with developers - so have more useful information about the profitable deals.
          </p>
```
исправь везде